helper/metricsutil/metricsutil.go

```diff
@@ -38,7 +40,14 @@ func FormatFromRequest(req *logical.Request) string {
 	if len(acceptHeaders) > 0 {
 		acceptHeader := acceptHeaders[0]
 		if strings.HasPrefix(acceptHeader, OpenMetricsMIMEType) {
-			return "prometheus"
+			return PrometheusMetricFormat
+		}
+
+		// Look for prometheus accept header
+		for _, header := range acceptHeaders {
+			if strings.Contains(header, PrometheusMetricFormat) {
```

Comment:
Guessing that this should be `PrometheusSchemaMIMEType` here? Also can we add a test for this?

Reply:
done